blueprints.csv

```diff
@@ -0,0 +1,30 @@
+﻿id,name
+128673330,Weapon_LightWeight
+128673345,Weapon_ShortRange
+128673500,Weapon_Overcharged
+128673640,Armour_HeavyDuty
+128673657,Engine_Dirty
+128673660,Engine_Reinforced
+128673667,Engine_Tuned
+128673685,FSD_FastBoot
+128673694,FSD_LongRange
+128673700,FSD_Shielded
+128673705,HullReinforcement_Advanced
+128673715,HullReinforcement_HeavyDuty
+128673730,PowerDistributor_HighCapacity
+128673737,PowerDistributor_HighFrequency
+128673765,PowerPlant_Boosted
+128673772,PowerPlant_Stealth
+128673795,ShieldBooster_Thermic
+128673805,ShieldCellBank_Rapid
+128673810,ShieldCellBank_Specialised
+128673820,ShieldGenerator_Kinetic
+128673825,ShieldGenerator_Optimised
+128673835,ShieldGenerator_Reinforced
+128673842,ShieldGenerator_Thermic
+128682083,Weapon_Sturdy
+128731526,Misc_LightWeight
+128739090,Weapon_Efficient
+128740134,Sensor_LongRange
+128740138,Sensor_WideAngle
+128740153,Sensor_Expanded
```

Comment:
My understanding is that there is a separate blueprint for each grade and that each has its own ID. Linking EDDI's definitions (which may also be imperfect) for reference: https://github.com/EDCD/EDDI/blob/develop/DataDefinitions/Blueprint.cs

Reply:
thanks for the observation, I hadn't noticed each grade had its own id

*grumbles incohernetly*

Reply:
ugh it gets worse 

not only does each blueprint grade have its own id, but it also varies depending on the module you apply it to, so weapon-efficient has different IDs for multicannon,beamlaser etc.